packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -286,7 +286,8 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	nearbyAppeals: new ListAddMoreQuestion({
-		title: 'Nearby appeal added to the case',
+		pageTitle: 'Nearby appeal added to the case',
+		title: 'n/a',
```

Comment:
feel as if we should make this optional on question class

Reply:
Yeah I thought the same but didn't want to pick up re working all the tests as part of this with the end of sprint comiog up